doc/lsst.analysis.tools/faqs.rst

```diff
@@ -48,3 +48,27 @@ in the pipeline YAML.
 The first line calls the action defined in python, after that the specific configuration is set in the
 pipeline YAML. This is a good balance between defining the entire action in YAML and having duplicate actions 
 defined in the python with only very mild differences.
+
+Can analysis tools put things in the butler?
+--------------------------------------------
+
+Short answer, yes! Long answer, while the tools themselves do not put things in the butler the tasks in the
+pipelines contained in analysis tools can. As an example lets look at the plots and metrics calculated from
+matching to the reference catalogues. First there is a task that matches the input data to the reference 
+catalogue. This then saves in the butler the matched output so that it can be read in later by multiple
+atools and also used in a notebook or script if further QA is requried. After the matched catalogue is made 
+the plotting/metric/calculation atools read it in, then act on it. The atools themselves do not butler put 
+the metric/plot/calculated data but return them to the task that ran them which then puts them in the right
+place. Most of this is hidden from the user and the atools handle putting plots and metrics themselves.
```

Comment:
This last sentence is a little confusing to me - are you using "atools" here to refer to "an analysis tool" or "analysis_tools"? If the former, as in the previous sentence, then I don't think this is accurate.

I'd recommend reserving "atools" for referring to individual analysis tools, and writing out the full "analysis tools" when referring to the complete package.